src/reducers/versions.tsx

```diff
@@ -89,6 +91,7 @@ export type VersionFile = {
 
 type VersionEntry = {
   depth: number;
+  file: VersionFile | null;
```

Comment:
We've had a lot of trouble in the past dealing with fields like this that may or may not be populated with an object. This is an indication to me that `file` does not belong here. Can we use another mapping?